guides/release-notes.md

```diff
@@ -8,6 +8,30 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 
 ---
 
+### Migration of frontend APIs
+
+In an effort to streamline our APIs we have migrated our frontend modules to contain the `-frontend` suffix. Legacy APIs were mostly divided between frontend and backend functionality and while the backend modules were indicated by a `-backend` suffix in the module name, the frontend modules had no indication.
+
+With the introduction of [universal modules](https://www.wix.com/velo/reference/api-overview/api-versions#api-overview_api-versions_universal-modules) we have now migrated the legacy frontend modules to clearly indicate that they are frontend only.
+
+You don't need to take any action at this time. As always, we'll continue to support the legacy APIs so your existing code will continue to work as it does today.
```

Comment:
Maybe also worth mentioning that the new docs stand in for the old docs? @talyaro wdyt?